### ISSUE TITLE
Hard coded ExternalConfiguration path in ApiClient.php::callAuthenticationHeader

### ISSUE THREAD

**Issue by SYip** [CyberSource/cybersource-rest-client-php#11]
``` 
    public function callAuthenticationHeader($method, $postData, $resourcePath)
    {
        require_once './Resources/ExternalConfiguration.php';
        $ExternalConfigurationObj = new ExternalConfiguration();
        $merchantConfig = $ExternalConfigurationObj->merchantConfigObject();

```

It assumes that there will be a Resources directory under project root and the path is not configurable.

The ExternalConfiguration should be pass as function params rather than requiring it in callAuthenticationHeader.

**Comment by damienwebdev** [comment]
@brianmc @gnongsie can we get a response back on this issue? This just seems like necessarily bad implementation code. I can submit a PR if needed, but this is currently a major blocker for a project of mine.

**Comment by gnongsie** [comment]
Hi,

Apologies for this. I'll take a look at this as soon as I can. Please be patient while I investigate this.

**Comment by gnongsie** [comment]
Hi,
I've provided the fix for this. It will entail a few changes to the samples but the changes are small.

A pull request will be sent shortly.

**Comment by gnongsie** [comment]
Hi,
The fix for this has been released as part of v0.0.6.

Thank you for bringing this to our notice. I will be closing this issue for now. 

Feel free to open a new issue for any further problems.
